CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixed
 
 - align IaC description panel inner indentations
+- update dependencies
```

Comment:
do we really want/need to inform users about that?...

Reply:
dunno. I'm open to removing it ;)